resources/helm/v1.1/istio/files/injection-template.yaml

```diff
@@ -278,27 +320,22 @@ containers:
     failureThreshold: {{ annotation .ObjectMeta `readiness.status.sidecar.istio.io/failureThreshold` .Values.global.proxy.readinessFailureThreshold }}
   {{ end -}}
   securityContext:
-    {{- if .Values.global.proxy.privileged }}
-    privileged: true
-    {{- end }}
-    {{- if ne .Values.global.proxy.enableCoreDump true }}
-    readOnlyRootFilesystem: true
-    {{- end }}
-    {{ if eq (annotation .ObjectMeta `sidecar.istio.io/interceptionMode` .ProxyConfig.InterceptionMode) `TPROXY` -}}
+    allowPrivilegeEscalation: {{ .Values.global.proxy.privileged }}
     capabilities:
+      {{ if eq (annotation .ObjectMeta `sidecar.istio.io/interceptionMode` .ProxyConfig.InterceptionMode) `TPROXY` -}}
       add:
       - NET_ADMIN
-    runAsGroup: 1337
-    {{ else -}}
-    capabilities:
+      {{- end }}
       drop:
-      - KILL
-      - SETUID
-      - SETGID
-      - MKNOD
-    {{ if .Values.global.sds.enabled }}
+      - ALL
```

Comment:
@luksa I am not 100% sure the SCC controller will be smart enough to not overwrite this - `ALL` should of course cover necessary restrictions, but it might be looking for specific caps?

Reply:
I added them back